src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -219,13 +219,15 @@ def _get_default_dns_prefix(name, resource_group_name, subscription_id):
 
 
 # pylint: disable=too-many-locals
-def store_acs_service_principal(subscription_id, client_secret, service_principal,
+def store_acs_service_principal(subscription_id, client_secret, service_principal, aad_session_key,
```

Comment:
Is this a new parameter of some command ? If yes, could you please also add argument definition such as help message for it ?